src/pages/LogIn.js

```diff
@@ -52,36 +82,51 @@ const LogIn = () => {
   return (
     <>
       <Heading1 content="Log in to your account here" />
-      <form
-        style={{
-          display: "flex",
-          flexDirection: "column",
-        }}
+      <StyledForm
         onSubmit={handleSubmit}
       >
         <p style={{ color: "red" }}>{state.errMsg && state.errMsg}</p>
-        <label style={{ width: "100px" }}>
-          Email
-          <input
+        <FormWrapper>
+          <StyledLabel for="email">Email</StyledLabel>
+          <StyledInput 
             name="email"
             onChange={handleChange}
             placeholder="alex@gmail.com"
             type="text"
             value={state.email}
           />
-        </label>
-        <label style={{ width: "100px" }}>
-          Password
-          <input
+
+          <StyledLabel for="password">Password</StyledLabel>
+          <StyledInput 
             name="password"
             onChange={handleChange}
             placeholder="$eCUre"
             type="password"
             value={state.password}
           />
-        </label>
-        <button style={{ width: "100px" }}>Login</button>
-      </form>
+          {/* <label style={{ width: "100px" }}>
```

Comment:
Styling looking good. I think we can chuck this commented out code now that we have better styles.